src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/DurableTaskStep.java

```diff
@@ -102,17 +104,15 @@ public boolean isReturnStatus() {
 
     public abstract static class DurableTaskStepDescriptor extends StepDescriptor {
 
-        public static final String defaultEncoding = "UTF-8";
-
         public FormValidation doCheckEncoding(@QueryParameter boolean returnStdout, @QueryParameter String encoding) {
+            if (encoding.isEmpty()) {
```

Comment:
`StringUtils.isBlank()`? otherwise the behavior will differ from `Util.fixEmpty`
🐜

Reply:
This is a web method; the text field may be blank but not missing.